docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,310 @@ sidebar_position: 18
 
-You can also run the occlum enclave either from host VM or from your own docker image:
+2. (Optional) If you don't want to accept the defaults, update the `vault.parameters.json` file with the following values. These parameters have default values and in most cases you won't need to make any updates.
 
-```
-tar xvf uid2-operator.tar.gz
-cd uid2-operator
-occlum run /bin/launcher
-```
+    Parameter | Description |
+   | :--- | :--- |
+   | `operatorIdentifier` | The name of the managed identity that will launch the container.<br/>Default: `uid-operator`. |
+   | `operatorKeyName` | The operator key secret name.<br/>Default: `uid-operator`. |
+
+   (**GWH_YS_11 a request. Could we use different default values for `operatorIdentifier` and `operatorKeyName`? I appreciate this might be a technical change and the answer might therefore be no. But I think it would be clearer if we didn't use the same default value for different settings. For example, operatorKeyName could be `uid-operator-key`. Commented same in Word file.**)
+
+3. Run the following command to trigger the deployment:
+
+   ```
+   az deployment group create --name vault --resource-group {RESOURCE_GROUP_NAME} --parameters vault.parameters.json  --template-file vault.json
+   ```
+
+### Set Up the VPC Network
+
+The next step is to set up the VPC network.
+
+The following diagram illustrates the virtual private cloud that hosts a UID2 Private Operator in Microsoft Azure.
+
+![VPC Network](images/operator-azure-drawio.png)
+
+Follow these steps:
+
+1. (Optional) If you don't want to accept the defaults, update the `vnet.parameters.json` file with the following values. These parameters have default values and in most cases you won't need to make any updates.
+
+    Parameter | Description |
+   | :--- | :--- |
+   | `vnetName` | The virtual network name.<br/>Default: `unified-id-network` |
+   | `computeSubnetName` | The name of the subnet that runs the UID2 Operator.<br/>Default: `unified-id-subnet-operators` |
+   | `gatewaySubnetName` | The name of the subnet that runs the UID2 Gateway.<br/>Default: `unified-id-subnet-gateway` |
+   | `VnetAddressPrefix` | The vnet address prefix.<br/>Default: `10.0.0.0/20` |
+   | `computeSubnetPrefix` | The vnet address prefix of the subnet that is delegated to run the UID2 Operator.<br/>Default: `10.0.0.0/24` |
+   | `gatewaySubnetPrefix` | The vnet address prefix of the subnet that runs the UID2 Gateway.<br/>Default: `10.0.0.0/28` |
+
+2. Run the following command to trigger the deployment:
+
+   ```
+   az deployment group create --name vnet --resource-group {RESOURCE_GROUP_NAME} --parameters vnet.parameters.json  --template-file vnet.json
+   ```
+
+### Complete the UID2 Private Operator Setup
+
+The next step is to bring up multiple Azure Container Instances (ACIs) in the VPC sub network that you created.
```

Comment:
Should this be subnetwork?

Reply:
Per research seems reasonable. Updated. Thx much.